scripts/asm-installer/install_asm

```diff
@@ -1079,6 +1099,20 @@ EOF
   WORKLOAD_POOL="${PROJECT_ID}.svc.id.goog"
 }
 
+validate_revision_label() {
+  # Matches DNS label formats of RFC 1123
+  local DNS1123LabelMaxLength; DNS1123LabelMaxLength=63
+  local DNS1123LabelFmt; DNS1123LabelFmt="[a-zA-Z0-9]([-a-zA-Z0-9]*[a-zA-Z0-9])?"
```

Comment:
Let's use UPPER_SNAKE_CASE to follow convention, and add `readonly VAR_NAME` for these after setting the value.